### PR TITLE
54 allow disabling commands

### LINKED ISSUE #54: Allow admins to enable/disable certain commands
Botomir-admin should be able to restrict certain commands from being permitted (ie. reddit command), to have better control over the bot. Admin's should also be able to enable these commands once disabled. All commands should come enabled by default.

### PR DESCRIPTION
**Pull request by MarshallAsch**
Closes #54 

Commands can now be enabled or disabled. 

Currently the following commands are marked as always enabled and can not be dissabled:
- `enable-command`
- `disable-command`
- `help`

### CODE REVIEW

File: lib/commands/admin/disableCommand.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const source = require('rfr');
+
+const { sendMessage } = source('lib/utils/util');
+const { Settings } = source('lib/database/models/settings');
+
+function addMemeSubCommand(message, args) {
```
Comment (SoorajModi) [inline lib/commands/admin/disableCommand.js:6]: Change the name of the function to disableCommand

File: lib/commands/admin/enableCommand.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const source = require('rfr');
+
+const { sendMessage } = source('lib/utils/util');
+const { Settings } = source('lib/database/models/settings');
+
+function addMemeSubCommand(message, args) {
```
Comment (SoorajModi) [inline lib/commands/admin/enableCommand.js:6]: Change name of function to enableCommand

File: lib/commands/admin/disableCommand.js (hunk 1 above)
Comment (MarshallAsch) [reply lib/commands/admin/disableCommand.js:6]: ooops copy paste

File: lib/commands/admin/enableCommand.js (hunk 2 above)
Comment (MarshallAsch) [reply lib/commands/admin/enableCommand.js:6]: ooops copy paste

**Pull request merged**